Java/E-montazysta/src/main/java/com/emontazysta/model/dto/EmploymentDto.java

```diff
@@ -1,21 +1,27 @@
 package com.emontazysta.model.dto;
 
+import com.emontazysta.validation.IsAfter;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.time.LocalDate;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
 
 @Data
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
+@IsAfter(message = "Date of dismiss must be after date of employment", startDateFieldName = "dateOfEmployment", endDateFieldName = "dateOfDismiss")
 public class EmploymentDto {
 
     private Long id;
-    private LocalDate dateOfEmployment;
-    private LocalDate dateOfDismiss;
+    @NotNull(message = "Date of employment cannot be empty")
+    private LocalDateTime dateOfEmployment;
+    private LocalDateTime dateOfDismiss;
+    @NotNull(message = "Company id cannot be empty")
```

Comment:
```suggestion
```
I think that it will be set based on data from token.

Reply:
I think here it's not appropriate. Current `companyId` is calculated base on employments.

Reply:
Yeah, you are right!